components/feature/addons/src/test/java/mozilla/components/feature/addons/ui/ExtensionsTest.kt

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.addons.amo.mozilla.components.feature.addons.ui
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.feature.addons.ui.translate
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.Locale
+
+@RunWith(AndroidJUnit4::class)
+class ExtensionsTest {
+
+    @Test
+    fun translate() {
```

Comment:
nit: maybe a test case for fallback option?

Reply:
👍

Reply:
Fixed!